### PR TITLE
add support for tweakers

### PR DESCRIPTION
**Pull request by bakzkndd**

### CODE REVIEW

File: websites/T/tweakers/presence.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    } else 
+      presenceData.details = "Scrollt nieuws artikelen...";
+    
+  } else if (document.location.href.includes("reviews/")) {
+    if(document.location.href.endsWith(".html")) {
+      presenceData.details = "Leest een review...";
+      presenceData.state = `Review: '${document.location.pathname
+        .split("/")[3]
+        .replace(".html", "")}'`;
+    } else if(document.location.pathname.split("/")[2] && document.location.pathname.split("/")[2] == "reviews") {
```
Comment (Slowlife01) [inline websites/T/tweakers/presence.ts:25]:
```suggestion
    } else if (document.location.pathname.split("/")[2] && document.location.pathname.split("/")[2] === "reviews") {
```

File: websites/T/tweakers/presence.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  } else 
+    presenceData.details = "Bezoekt de tweakers pagina...";
+  
+
+  if (presenceData.details == null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else 
+    presence.setActivity(presenceData);
+
```
Comment (Slowlife01) [inline websites/T/tweakers/presence.ts:121]:
```suggestion
  if (!presenceData.details) {
    presence.setTrayTitle();
    presence.setActivity();
  } else presence.setActivity(presenceData);
```

File: websites/T/tweakers/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    ],
+    "url": "tweakers.net",
+    "description": {
+      "nl": "Tweakers is de grootste elektronica- en technologiewebsite van Nederland en België.",
+      "en": "Tweakers is the biggest electronics and technology website in the Netherlands and Belgium."
+    }
+  }
+  
+  
+
```
Comment (Slowlife01) [inline websites/T/tweakers/dist/metadata.json:31]:
```suggestion
```

File: websites/T/tweakers/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Tweakers",
+    "author": {
+      "name": "bakzkndd#2819",
```
Comment (Slowlife01) [inline websites/T/tweakers/dist/metadata.json:5]:
```suggestion
      "name": "bakzkndd",
```

File: websites/T/tweakers/presence.ts [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    presenceData.details = "Bekijkt zijn/haar karma...";
+   else if (document.location.href.includes("instellingen")) 
+    presenceData.details = "Bekijkt instellingen...";
+   else if (document.location.pathname && document.location.pathname.length > 2) {
+    presenceData.details = "Bladert...";
+    presenceData.state = `Op pagina '${document.location.pathname
+      .split("/")[1]}'`;
+  } else 
+    presenceData.details = "Bezoekt de tweakers pagina...";
+
```
Comment (Slowlife01) [inline websites/T/tweakers/presence.ts:114]:
```suggestion
  } else presenceData.details = "Bezoekt de tweakers pagina...";
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Please run prettier as well

**Review by Slowlife01 — COMMENTED** [review]
Proof of it working

**Comment by bakzkndd** [comment]
image
image
image

a few screenshots showing it in action

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**